packages/simple-hub/src/start-dev-server.ts

```diff
@@ -13,18 +13,18 @@ async function setupGanacheAndContracts() {
 }
 
 async function startLocalFirebaseServer() {
-  const server = new FirebaseServer(5555, 'localhost');
```

Comment:
Should this be deleted for now?

Reply:
It would be nice to get it working, so that we can use a local firebase server. But it was not my priority at the time.